src/test/java/com/google/sps/data/ChangeControllerServletTest.java

```diff
@@ -0,0 +1,58 @@
+package com.google.sps.data;
+
+import java.io.IOException;
+import java.util.Optional;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.servlets.ChangeControllerServlet;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+
+public class ChangeControllerServletTest {
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+  private final DatastoreClientInterface datastoreClient = new DatastoreClient();
+
+  @Mock
+  HttpServletRequest request;
+
+  @Mock
+  HttpServletResponse response;
+
+  @Spy
+  ChangeControllerServlet servlet = new ChangeControllerServlet();
```

Comment:
Why is this the only one not instantiated in setUp()?

Reply:
it is a spy, so it has to be instantiated when its declared